source/Server.Extensibility/HostServices/Model/Projects/ProjectConnectivityPolicy.cs

```diff
@@ -9,9 +9,9 @@ public enum SkipMachineBehavior
         SkipUnavailableMachines
     }
 
-    public class ProjectConnectivityPolicy
+    public class DeploymentConnectivityPolicy
     {
-        public ProjectConnectivityPolicy()
+        public DeploymentConnectivityPolicy()
```

Comment:
We have renamed `ProjectConnectivityPolicy` as it no longer applies to the project. It's more likely this would apply more to the deployment at this point.